lake/root.go

```diff
@@ -333,6 +333,9 @@ func (r *Root) CreatePool(ctx context.Context, name string, sortKey order.SortKe
 	if thresh == 0 {
 		thresh = data.DefaultThreshold
 	}
+	if len(sortKey.Keys) > 1 {
+		return nil, errors.New("multiple pool keys not supported")
+	}
```

Comment:
This is all we need for #4596. I think we should leave everything else as is.

Reply:
My reasoning here is that we'll have to revert all the other changes when we add support for multiple pool keys.

Reply:
I can go along with this (in fact my first revision of this pr was similar) with the caveat that we should keep the help document changes and also if we do it this way the server will have to be adjusted so this is 400 error.

Reply:
I also think once we decide to add support for multiple pool keys we will have to augment order.SortKeys so that each key can have a different sort order, so we'll have to make changes on everything this pr touches anyway.